upload/admin/view/template/extension/shipping/zasilkovna_orders.twig

```diff
@@ -169,6 +203,11 @@
                   <td class="text-left">{{ order.order_status }}</td>
                   <td class="text-left">{{ order.total }}</td>
                   <td class="text-left">{% if order.is_cod %} {{ text_yes }} {% else %} {{ text_no }} {% endif %} </td>
+                  <td class="text-left">
+                    <div class="grid-input-pair">
+                      <input class="grid-text-input" type="number" step="0.0001" value="{{ order.weight }}" name="weight[{{ order.order_id }}]"><span>kg</span>
```

Comment:
Pokud to nějak nekopíruje chování eshopu, použil bych step 0.001

Reply:
Prohlížeč pak ale nedovolí zadat menší hodnotu

Reply:
0 i 1 g nastavit dovolí, ne?